src/Google/Service/Logging.php

```diff
@@ -16,7 +16,7 @@
  */
 
 /**
- * Service definition for Logging (v2).
+ * Service definition for Logging (v2beta1).
```

Comment:
Argh. Good question. Hold off on this, then, until we support versioned APIs (#44). We really need to.

Thanks for the catch.